api/provider_api.md

```diff
@@ -24,22 +24,72 @@
 ## Test Drive
 
 Once the above are defined (and stuffed as environment variables),
-you are ready to user the [Provider API](https://www.adobe.io/apis/experienceplatform/events/ioeventsapi.html#/Providers/getProvidersByConsumerOrgId)
+you are ready to use API, confer its [`swagger`/`OpenApi` documentation](https://www.adobe.io/apis/experienceplatform/events/ioeventsapi.html)
```

Comment:
you are ready to use **the** API, **refer to** its ...

Reply:
done https://github.com/AdobeDocs/adobeio-events/pull/77/commits/dbb81a47871472b96256d42c87871cc1b87a4b9f